packages/components/src/slot-fill/bubbles-virtually/slot-fill-provider.tsx

```diff
@@ -66,12 +60,7 @@ function createSlotRegistry(): SlotFillBubblesVirtuallyContext {
 			return;
 		}
 
-		slot.fillProps = fillProps;
-		const slotFills = fills.get( name );
-		if ( slotFills ) {
-			// Force update fills.
-			slotFills.forEach( ( fill ) => fill.rerender() );
-		}
+		slots.set( name, { ref, fillProps } );
```

Comment:
This is the key change in this PR! Instead of silently updating a field with `slots.get( name ).fillProps = fillProps`, we call `slots.set`. That sets a new value and triggers calling all `observableMap` listeners, leading to automatic rerendering of all fills.